apiary.apib

```diff
@@ -782,6 +782,8 @@ Get details for a single contact.
                         + id: `40157f9d-1322-4f2e-b245-188efdc52cc2` (string)
             + language: `en` (string)
             + payment_term (PaymentTerm, nullable)
+            + invoicing_preferences (object)
+                + electronic_invoicing_address (string, nullable)
```

Comment:
Q: should this be `electronic_invoice_address`?

Reply:
Aren't we using electronic_invoicing_address as domain term everywhere? I copy pasted the term from the codebase 😉

Reply:
Indeed, somehow I thought we were using the one I asked 🙈